include/picongpu/param/physicalConstants.param

```diff
@@ -40,24 +45,35 @@ namespace picongpu
         constexpr float_64 Z0_SI = MUE0_SI * SPEED_OF_LIGHT_SI;
 
         /** reduced Planck constant
+         *
          * unit: J * s
+         *
+         * 2022 CODATA value, https://physics.nist.gov/cgi-bin/cuu/Value?hbar
          */
-        constexpr float_64 HBAR_SI = 1.054571800e-34;
+        constexpr float_64 HBAR_SI = 1.054571817e-34;
 
         // Electron properties
-        /** unit: kg */
-        constexpr float_64 ELECTRON_MASS_SI = 9.109382e-31;
-        /** unit: C */
-        constexpr float_64 ELECTRON_CHARGE_SI = -1.602176e-19;
+        //! unit: kg,   2022 CODATA value, https://physics.nist.gov/cgi-bin/cuu/Value?me
+        constexpr float_64 ELECTRON_MASS_SI = 9.1093837139e-31;
+        //! unit: C,    2022 CODATA value, https://physics.nist.gov/cgi-bin/cuu/Value?e
+        constexpr float_64 ELECTRON_CHARGE_SI = -1.602176634e-19;
 
         /* atomic unit for energy:
-         * 1 Rydberg = 27.21 eV --> converted to Joule
+         * 2 Rydberg = 27.21 eV --> converted to Joule
          */
         constexpr float_64 ATOMIC_UNIT_ENERGY = 4.36e-18;
 
+        /** Rydberg energy, unit: eV
+         *
+         * 2022 CODATA value, https://physics.nist.gov/cgi-bin/cuu/Value?rydhcev
+         */
+        constexpr float_X RYDBERG_ENERGY = 13.605693122990_X; // eV
```

Comment:
Why is this stored as `float_X` and not as `float_64`?

Reply:
good catch, fixed